best_practices.rst

```diff
@@ -238,7 +238,7 @@ Use Dependency Injection to Get Services
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you extend the base ``AbstractController``, you can only get access to the most
-common services (e.g ``twig``, ``router``, ``doctrine``, etc.), directly from the
+common services (e.g. ``twig``, ``router`` and ``doctrine``), directly from the
```

Comment:
I'd say this change is wrong because the AbstractConroller gives you access to lots of services ... and not only those 3. See:

https://github.com/symfony/symfony/blob/b004c3c6d120c4d16988c97bf01268adfe960cb1/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L83-L95